src/managed/CommonManaged.props

```diff
@@ -11,6 +11,8 @@
     <DisableImplicitFrameworkReferences>true</DisableImplicitFrameworkReferences>
     <IncludeSymbols>true</IncludeSymbols>
     <Serviceable>true</Serviceable>
+    <PackageLicenseUrl>https://github.com/dotnet/core-setup/blob/master/LICENSE.TXT</PackageLicenseUrl>
```

Comment:
Looks like it comes from https://github.com/dotnet/buildtools/blob/3309085641d2fe6a6193915dd89ffbb159dad9fb/src/Microsoft.DotNet.Build.Tasks.Packaging/src/PackageFiles/Packaging.targets#L40-L46